app/src/main/java/db/dto/QuizDTO.kt

```diff
@@ -0,0 +1,15 @@
+package db.dto
```

Comment:
duo's are data transfer objects. In this case transfer means from backend to app. Why are they part of db package then?